do-dont/src/InstructionsCard.ts

```diff
@@ -1,48 +1,30 @@
-import { LitElement, html, TemplateResult } from 'lit';
+import { LitElement, html, TemplateResult, CSSResultGroup } from 'lit';
 import { property } from 'lit/decorators.js';
 import { InstructionsCardStyles } from './InstructionsCard.styles';
 
-type Icon = string | TemplateResult | ((color: string) => TemplateResult);
+type Icon = string | TemplateResult;
 
 export abstract class InstructionsCard extends LitElement {
-  static styles = InstructionsCardStyles;
-
-  @property()
-  color: string;
-
-  /**
-   * Takes string or lit-html template. To render correct color automatically wrap in a function with `color` as a param.
-   * @type string | `lit-html template` | `(color: string) => lit-html template`
-   */
-  @property()
-  icon: Icon;
+  static styles: CSSResultGroup = InstructionsCardStyles;
 
   @property()
   label: string;
 
-  render() {
-    const icon =
-      typeof this.icon === 'function' ? this.icon(this.color) : this.icon;
+  icon: Icon = '';
 
+  render() {
     return html`
-      <div class="container">
-        <div class="component-container">
+      <div part="container" class="container">
```

Comment:
Not sure 100% but I think we don't need class here something like this should do the job
```
[part=container] {
...
}
```